test/etcetera/etcd_v2_test.exs

```diff
@@ -0,0 +1,164 @@
+defmodule EtceteraTest do
+  use ExUnit.Case
+
+  alias Etcetera.EtcdV2
+
+  doctest EtcdV2
```

Comment:
Obviously this might change going forward but maybe temporarily note in the README that these will fail without an an accessible Etcd server up (with auth)?